bubble_sort.py

```diff
@@ -0,0 +1,14 @@
+def bubbleSort(arr):
+    for i in range(len(arr)):
+        swapped = False
+        for j in range(0, len(arr) - i - 1):
+            if(arr[j]>arr[j+1]):
+                temp = arr[j]
+                arr[j] = arr[j+1]
+                arr[j+1] = temp
+                swapped = True
+                
+        if not swapped:
+            break
+  
```

Comment:
There are too many extra lines at the end.

Reply:
Okay I'm on it!